patches/minecraft/net/minecraft/world/item/UseAnim.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/item/UseAnim.java
++++ b/net/minecraft/world/item/UseAnim.java
+@@ -9,5 +_,6 @@
+    SPEAR,
+    CROSSBOW,
+    SPYGLASS,
+-   TOOT_HORN;
++   TOOT_HORN,
++   CUSTOM;
```

Comment:
Could you please clarify for me when this `UseAnim` value should be used by modders?

Reply:
It is to avoid vanilla trying to "animate" items with custom usage animation.

Reply:
Please do so, especially so it is clearly differentiated versus `UseAnim.NONE`.